06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,221 @@
+/* global FileReader */
+/* global InvertedIndex */
+/**
+ * @class InvertedIndex
+ * @classdesc blah blah
+ */
+/* eslint-disable */
+class InvertedIndex {
+  /* eslint-enable */
+  /**
+   * * @constructor
+   * initialises the class base properties
+   */
+  constructor() {
+    this.indicies = {};
+    this.indexedFiles = {};
+    this.uploadedFiles = {};
+  }
+  /**
+   * @createIndex method
+   * @param {fileName} fileName
+   * @param {fileContent} fileContent
+   * @returns {boolean}
+   * create index of the fileName
+   */
```

Comment:
All your JSDoc for this file are not properly done. Please review all. Ensure that your @param(s) and @return(s) are properly described also. Use the right dataTypes also.

You probably need to research JSDocs to have a better understanding